app/views/teams/show.html.haml

```diff
@@ -18,7 +18,7 @@
 
       %div.container.title-container
       - if @team.description == nil || @team.description == ""
-        %h6 Tesssssssstss  tststs tst stststst stststs tststs Tesssssssstss  tststs tst stststst stststs tststs Tesssssssstss  tststs tst stststst stststs tststs Tesssssssstss  tststs tst stststst stststs tststs Tesssssssstss  tststs tst stststst stststs tststs
```

Comment:
Was this line for testing purposes?

Reply:
yep, to see if the string would continue on a single line, or wrap around to make a paragraph. it made a paragraph